src/main/java/org/infernalstudios/miningmaster/init/MMSounds.java

```diff
@@ -29,7 +29,7 @@ public class MMSounds {
 
     // GEM FORGE
     public static final RegistryObject<SoundEvent> GEM_FORGE_COOK = registerSound("gemforge.cook");
-
+    public static final RegistryObject<SoundEvent> GEM_FORGE_COMPLETE = registerSound("gemforge.complete");
```

Comment:
is this a bugfix or what?

Reply:
yes, when the Gem Forge finished the subtitle would say "eye of ender attaches"